packages/mask/src/plugins/Avatar/Application/NFTInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Link, Stack, Typography } from '@mui/material'
+import { makeStyles, useStylesExtends } from '@masknet/theme'
+import { ApplicationIcon } from '../assets/application'
+import { LinkIcon } from '../assets/link'
+import { resolveOpenSeaLink } from '@masknet/web3-shared-evm'
+import { formatTokenId } from '../utils'
+import { useI18N } from '../locales/i18n_generated'
+
+const useStyles = makeStyles()(() => ({
+    root: {},
+    nft: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+}))
+interface NFTInfoProps extends withClasses<'root'> {
```

Comment:
```suggestion
interface NFTInfoProps {
```

withClasses should be only appears for general components.